detekt/defs.bzl

```diff
@@ -78,65 +141,37 @@ def _impl(ctx):
         arguments = [java_arguments, detekt_arguments],
     )
 
-    return [DefaultInfo(files = depset(action_outputs))]
+    final_result = ctx.actions.declare_file(ctx.attr.name + ".sh")
+    ctx.actions.run_shell(
+        inputs = [execution_result],
+        outputs = [final_result],
+        command = "exit_code=$(cat %s); echo -e \"#!/bin/bash\\n\\nexit $exit_code\" > %s" % (execution_result.path, final_result.path),
```

Comment:
This will make `rules_detekt` incompatible with Windows, I think it's ok for now but a comment hinting that would be great, maybe later someone needing Windows support will rewrite this in cross-platform way :)